### ISSUE TITLE
Public header menu (KB) ignores theme change (light to dark) 

### ISSUE THREAD

**Issue by Suhopoljac** [zammad/zammad#5263]
### Used Zammad Version

6.3.1

### Environment

- Installation method: any
- Operating system (if you're unsure: `cat /etc/os-release` ): any
- Database + version:  any
- Elasticsearch version: any
- Browser + version: MS Edge, v. 126.0.2592.87 (official build) (64-bit)

### Actual behaviour

When looking at the "Public header" field in the knowledge base settings, changing the theme from a light to a dark mode does not affect these two fields; they remain white in any mode. 

### Expected behaviour

When looking at the "Public Header" field in the knowledge base settings, changing the theme from a light mode to a dark mode should reflect the selected mode, so if the light mode is selected, then the field should be shown in white, if the dark mode, then tinted black.  

### Steps to reproduce the behaviour

1) Navigate to the KB settings in the admin settings.
2) Click through to the "Public Menu" tab
3) switch between the modes and enjoy the fireworks

https://github.com/user-attachments/assets/b478c269-a874-4d77-9047-9d69c2c0cdf6

### Support Ticket

Ticket#10157975

### I'm sure this is a bug and no feature request or a general question.

yes

**Comment by rolfschmidt** [comment]
@mantas is this really a bug? It looks like that the public menu links are generated based on the theme colors of the KB language.

https://github.com/zammad/zammad/blob/d3938ae1f9ec46e098db0892eff3af19be7225c5/app/assets/javascripts/app/views/knowledge_base/public_menu_manager.jst.eco#L17

works as designed?

**Comment by mantas** [comment]
@rolfschmidt yep, seems to be fireworks-by-design 😁  No way to solve this without full-blown dark mode in public KB...

I'll close this issue for now.
